hack/overlays/values.yaml

```diff
@@ -0,0 +1,5 @@
+#@data/values
+---
+namespace:
+clusterrole:
+  name:
```

Comment:
do we need to declare this file if we're not setting any default values?

Reply:
I tried to remove the values.yaml file and got an error, I'll take a look at the docs to see if it's required or not